curation_portal/views/variants.py

```diff
@@ -12,14 +12,13 @@ class VariantsView(APIView):
     def get(self, request, *args, **kwargs):  # pylint: disable=unused-argument
         variants = (
             Variant.objects.filter(
-                Q(reference_genome="GRCh37")  # TODO: Handle different reference genomes
-                & (
-                    Q(project__owners__id__contains=request.user.id)
-                    | Q(curation_assignment__curator=request.user)
-                )
+                Q(project__owners__id__contains=request.user.id)
+                | Q(curation_assignment__curator=request.user)
             )
-            .values_list("variant_id", flat=True)
+            .values_list("variant_id", "reference_genome")
```

Comment:
`flat=True` came out on purpose? IDK what that flag actually does, just making sure.